doc/source/tune/_tutorials/overview.rst

```diff
@@ -25,60 +21,36 @@ Take a look at any of the below tutorials to get started with Tune.
    :figure: /images/tune.png
    :description: :doc:`A walkthrough to setup your first Tune experiment <tune-tutorial>`
 
-.. raw:: html
-
-    </div>
-
-.. toctree::
-   :hidden:
-
-   tune-60-seconds.rst
-   tune-tutorial.rst
-
-
-User Guides
------------
-
-These pages will demonstrate the various features and configurations of Tune.
-
-.. raw:: html
-
-    <div class="sphx-glr-bigcontainer">
-
-.. customgalleryitem::
-   :tooltip: Tune User Guide
-   :figure: /images/tune.png
-   :description: :doc:`Tune User Guide <tune-usage>`
-
-.. customgalleryitem::
-   :tooltip: A simple guide to Population-based Training
-   :figure: /images/tune-pbt-small.png
-   :description: :doc:`A simple guide to Population-based Training <tune-advanced-tutorial>`
-
-.. customgalleryitem::
-   :tooltip: A guide to distributed hyperparameter tuning
-   :figure: /images/tune.png
-   :description: :doc:`A guide to distributed hyperparameter tuning <tune-distributed>`
 
 .. customgalleryitem::
    :tooltip: Tune's Scikit-Learn Adapters
    :figure: /images/tune-sklearn.png
-   :description: :doc:`Tune's Scikit-Learn Adapters <tune-sklearn>`
+   :description: :doc:`Tune's Scikit-Learn Adapters <tutorials/tune-sklearn>`
 
 .. customgalleryitem::
    :tooltip: Tuning PyTorch Lightning modules
    :figure: /images/pytorch_lightning_small.png
-   :description: :doc:`Tuning PyTorch Lightning modules <tune-pytorch-lightning>`
+   :description: :doc:`Tuning PyTorch Lightning modules <tutorials/tune-pytorch-lightning>`
 
 .. customgalleryitem::
    :tooltip: How to use Tune with PyTorch
    :figure: /images/pytorch_logo.png
-   :description: :doc:`How to use Tune with PyTorch <tune-pytorch-cifar>`
+   :description: :doc:`How to use Tune with PyTorch <tutorials/tune-pytorch-cifar>`
 
 .. customgalleryitem::
    :tooltip: Tuning XGBoost parameters.
    :figure: /images/xgboost_logo.png
-   :description: :doc:`A guide to tuning XGBoost parameters with Tune <tune-xgboost>`
+   :description: :doc:`A guide to tuning XGBoost parameters with Tune <tutorials/tune-xgboost>`
+
+.. customgalleryitem::
```

Comment:
Would it be possible to add this PBT guide and the distributed experiments guide to the Tutorials dropdown on the left bar?

Reply:
Good idea.